test/integration/policy_info_metric_test.go

```diff
@@ -196,23 +236,27 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test policy_governance_info metric
 		By("Checking the policy metric")
 		policyLabel := `policy="` + noncompliantPolicyName + `"`
 		Eventually(func() interface{} {
-			resp, _, err := common.GetWithToken(propagatorMetricsURL, strings.TrimSpace(metricsToken))
+			resp, _, err := common.GetWithToken(
+				propagatorMetricsURL,
+				strings.TrimSpace(metricsToken),
+			)
 			if err != nil {
 				return err
 			}
+
 			return resp
 		}, defaultTimeoutSeconds, 1).Should(common.MatchMetricValue(metricName, policyLabel, "1"))
 	})
 	AfterAll(func() {
-		common.OcHub("delete", "-f", compliantPolicyYaml, "-n", userNamespace)
-		common.OcHub("delete", "-f", noncompliantPolicyYaml, "-n", userNamespace)
-		common.OcHub("delete", "route", "-n", ocmNS, "-l", propagatorMetricsSelector)
-		common.OcHub("delete", "clusterrole", metricsRoleName)
-		common.OcHub("delete", "clusterrolebinding", roleBindingName)
-		common.OcHub("delete", "serviceaccount", metricsAccName, "-n", userNamespace)
-		common.OcHub("delete", "serviceaccount", noMetricsAccName, "-n", userNamespace)
-		common.OcHub("delete", "secret", metricsTokenName, "-n", userNamespace)
-		common.OcHub("delete", "secret", noMetricsTokenName, "-n", userNamespace)
-		common.OcHub("delete", "namespace", "policy-metric-test-compliant")
+		_, _ = common.OcHub("delete", "-f", compliantPolicyYaml, "-n", userNamespace)
```

Comment:
It might be good to check the errors on these after you add the `--ignore-not-found` flag to these commands.